views/rights.py

```diff
@@ -56,6 +61,7 @@ def get(self, request, *args, **kwargs):
                     ],
                 },
             }
+            logger.warning(f'Group does not exist slug = {slug}')
```

Comment:
Rephrase for clarity:
```suggestion
            logger.warning(f'Group with slug \'{slug}\' does not exist')
```